apptest.go

```diff
@@ -75,6 +137,11 @@ func (a *AppSetup) InstallApps(ctx context.Context, apps []App) error {
 	return nil
 }
 
+// K8sClient returns a Kubernetes clienset for use in automated tests.
+func (a *AppSetup) K8sClient() kubernetes.Interface {
+	return a.k8sClient
+}
```

Comment:
The k8s client can be accessed for use in tests. This means only the apptest library is needed.